duckduckgo/Android
# Android: Allow the use of user certificates for nightly builds

Task/Issue URL: https://app.asana.com/0/414730916066338/1202500711374963/f

### Description
Allowed user certificates for internal builds.

### Steps to test this PR

- checkout this branch
- build the release versions for play (`./gradlew assemblePR`), fdroid (`./gradlew assembleFR`), internal (`./gradlew assembleIR`) 
- install each of them and have Charles proxy configured
- verify you can see traffic in Charles only for the internal build

### NO UI changes
